FreeRTOS/Test/CMock/smp/single_priority_timeslice/covg_single_priority_timeslice_utest.c

```diff
@@ -88,6 +92,38 @@ void vApplicationMinimalIdleHook( void )
 {
 }
 
+static void prvCreateStaticSMPTask( TaskHandle_t xTaskHandle,
```

Comment:
Why not move this function to smp_utest_common.c rather than duplicating?

Reply:
This is a good suggestion. Actually the function just setups the member variables which will be used in the test case. It is not a complete one.